scipy/signal/tests/test_ltisys.py

```diff
@@ -264,6 +264,15 @@ def test_basic(self):
         assert_allclose(bb[0], b, rtol=1e-13)
         assert_allclose(aa, a, rtol=1e-13)
 
+    def test_zero_order_round_trip(self):
+        # See Issue #5760
+        tf = (2, 1)
+        ss = tf2ss(*tf)
+        assert_equal(ss, [[0], [0], [0], [2]])
+
+        tf = ss2tf(*ss)
+        assert_equal(tf, [[[2, 0]], [1, 0]])
```

Comment:
Not entirely sure TBH.  One could argue that it is a bug because `2 / 1` is not the same technically as `2s / 1s`.  I do think nevertheless that that would be a separate issue.